docs/reference/migration/migrate_8_0/security.asciidoc

```diff
@@ -6,6 +6,41 @@
 //Installation and Upgrade Guide
 
 //tag::notable-breaking-changes[]
+.The file and native realms are now enabled unless explicitly disabled.
+[%collapsible]
+====
+*Details* +
+The file and native realms are now enabled unless explicitly disabled. If
+explicitly disabled, the file and native realms remain disabled at all times.
+
+Previously, the file and native realms had the following implicit behaviors:
+
+* If the file and native realms were not configured, they were implicitly disabled
+if any other realm was configured.
+
+* If no other realm was available because realms were not configured or
+explicitly disabled, the file and native realms were enabled, even if explicitly
+disabled.
```

Comment:
```suggestion
* If no other realm was available because realms were either not configured,
not perrmitted by license, or explicitly disabled, the file and native realms 
were enabled, even if explicitly disabled.
```